clinical-nightwatch-changes.txt

```diff
@@ -0,0 +1,2345 @@
+
```

Comment:
what is this file?

Reply:
Oh. These should not have been included. It was from when I was doing timing comparisons

Reply:
ah ok, makes sense.